rectools/models/nn/item_net.py

```diff
@@ -317,6 +327,11 @@ def from_dataset_schema(
         n_items = dataset_schema.items.n_hot
         return cls(n_factors, n_items, dropout_rate)
 
+    @property
```

Comment:
Let's also implement `out_dim` for `SumOfEmbeddingsConstructor` as out_dim of the first block. We might need it in the future.